pallets/xcm-transactor/src/lib.rs

```diff
@@ -148,28 +148,38 @@ pub mod pallet {
 	#[derive(Default, Clone, Encode, Decode, RuntimeDebug, PartialEq, scale_info::TypeInfo)]
 	pub struct RemoteTransactInfoWithMaxWeight {
 		/// Extra weight that transacting a call in a destination chain adds
+		/// Extra weight involved when transacting without DescendOrigin
 		pub transact_extra_weight: Weight,
-		/// Fee per weight in the destination chain
-		pub fee_per_second: u128,
 		/// Max destination weight
 		pub max_weight: Weight,
+		/// Whether we allow transacting through signed origins in another chain, and
+		/// how much extra cost implies
+		/// Extra weight involved when transacting with DescendOrigin
+		pub transact_extra_weight_signed: Option<Weight>,
```

Comment:
`transact_extra_weight_signed` and `transact_extra_weight` are mutually exclusive and we always use one of them, right?

Reply:
If so, why an `Option` on one and not the other?

Reply:
I will add comments